liveclasses/models.py

```
@@ -0,0 +1,64 @@
+
+    def get_meeting_url_teacher(self):
+        """Return the secure Jitsi meeting URL with JWT authentication."""
+
+        token = self.generate_jwt_token()
+        return f"https://8x8.vc/{settings.JITSI_APP_ID}/{self.meeting_id}?jwt={token.decode()}"
+
+    def get_meeting_url_student(self):
+        return f"https://8x8.vc/{settings.JITSI_APP_ID}/{self.meeting_id}"
+
+    def __str__(self):
+        return f"{self.topic} - {self.course.title} ({self.scheduled_at})"
```

Ruff (0.8.2)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `__str__` method should come before `save` method

(DJ012)